httpserver/service_node.cpp

```diff
@@ -316,6 +345,58 @@ void ServiceNode::attach_signature(
     }
 }
 
+void ServiceNode::perform_peer_test() {
+
+    // 1. Select the tester/testee pair
+
+    std::vector<sn_record_t> members_pk = swarm_->other_nodes();
+
+    members_pk.push_back(our_address_);
+
+    std::sort(members_pk.begin(), members_pk.end());
+
+    std::cout << "peers: " << std::endl;
```

Comment:
Probably best using the filterable `BOOST_LOG_TRIVIAL` interface?

Reply:
That's for debugging, will remove.